### ISSUE TITLE
Datadog bought Sqreen; figure that out

### ISSUE THREAD

**Issue by colinxfleming** [DARIAEngineering/dcaf_case_management#2687]
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**

https://www.datadoghq.com/blog/datadog-acquires-sqreen/

* **What feature or behavior is this required for?**

appsec

* **How could we solve this issue? (Not knowing is okay!)**

idk

* **Anything else?**

**Comment by colinxfleming** [comment]
FAQ https://assets.sqreen.com/website/2022-05_Sqreen-customer-transition_FAQ.pdf

**Comment by colinxfleming** [comment]
no nonprofit rate sad but true

**Comment by colinxfleming** [comment]
okay, so the figure to beat here is 31/server/month for stuff like sql injection protection and other stuff listed here: https://www.datadoghq.com/product/security-platform/application-security-monitoring/ I think we're going to do some shopping and see if we can beat it, and see where we're at after

**Comment by colinxfleming** [comment]
@xmunoz can I tag you in on doing this research?

**Comment by xmunoz** [comment]
Yep!

**Comment by xmunoz** [comment]
Alright, here are some alternatives:

[AWS WAF](https://aws.amazon.com/waf/)
>  With AWS WAF, you can create security rules that control bot traffic and block common attack patterns such as SQL injection or cross-site scripting (XSS). 

The integration effort for this would be medium to high, because we would need to setup a cloudfront distribution as well.
FAQS: https://aws.amazon.com/waf/faqs/
Pricing: https://aws.amazon.com/waf/pricing/
I think the cost of this would probably between free and ~$10/month, but it's hard to tell because of the ancillary services we would need to setup.

[Cloudflare WAF](https://www.cloudflare.com/waf/)
> Cloudflare offers powerful rulesets that stop threats including newly discovered zero days, as well as bypasses and attack variations using machine learning. Additionally, with granular custom rules, you can configure the WAF to protect against any threat or implement business-specific security policies.
The integration effort for this looks to be low to medium, seems to be a lot more turnkey than AWS.
Pricing: https://www.cloudflare.com/plans/#overview
Cost appers to be $20/month.

**Comment by colinxfleming** [comment]
I would prefer not to go pure AWS here - we've made it this far without having an AWS account and the longer that's true the better, so let's rule out FortiWeb, AWS WAF for that reason. If Bitmitigate doesn't have prices listed then that probably means out of our price range (and that's annoying).

ExpeditedWAF would nearly double our server bill; that's ugly but okay if we gotta.

I think Cloudlfare has the benefit of the price being right, and like you said, potential other very real bonuses (which hopefully work well with heroku hosting). Fastly is the other service in that weight class right? I took a gander at it - looks nice but looks like the floor is $50 + maybe a meter for basically the same stuff.

**Comment by colinxfleming** [comment]
@xmunoz if you think you can probably rig up Cloudflare I'll ask for clearance to spend the $$ and we can get it cooking this month or next, I think is my takeaway.

**Comment by xmunoz** [comment]
Yeah, that is what I was leaning towards as well. I just created an account and sent you an invite so you can add billing info at your leisure.

**Comment by xmunoz** [comment]
The next step is to migrate our nameservers to Cloudflare. Before doing this, I need to copy over all of the DNS records to Cloudflare.

**Comment by xmunoz** [comment]
I'm going to open a separate ticket for migrating to Cloudflare, since it's a lot of steps.
